lib/src/source/hosted.dart

```diff
@@ -1813,12 +1831,17 @@ class Advisory {
   Set<String> affectedVersions;
   List<String> aliases;
   String summary;
-  Advisory(this.id, this.affectedVersions, this.aliases, this.summary);
-}
+  String? pubDisplayUrl;
+  Advisory({
+    required this.id,
+    required this.affectedVersions,
+    required this.summary,
+    this.aliases = const [],
+    this.pubDisplayUrl,
+  });
 
-// TODO(https://github.com/dart-lang/pub-dev/issues/7471) Use `pub_display_url`
-// once this issue is resolved.
-String advisoriesDisplayUrl(String id) => 'https://github.com/advisories/$id';
+  String get displayUrl => pubDisplayUrl ?? id;
```

Comment:
Can we call this something different - since it is not a url, and to avoid confusion with `pubDisplayUrl`?

Maybe `displayHandle`

Reply:
Good idea.